documentation/example-session.md

```diff
@@ -7,38 +7,43 @@ const arc = new Arc({
   web3Provider: 'https://url.to.web3.node/',
 })
 
-// get an Observable list of DAOs
+// get an Observable for list of DAOs
 const daoObservable = arc.daos()
-// or just get the list itself
-const daos = await arc.daos().toPromise()
+// we get the first return item from the obervable
+const { first } = require('rxjs/operators')
+const daos = await arc.daos().pipe(first()).toPromise()
 
 // you can query for a particular DAO
 arc.daos({ name: 'mydao'})
 
-const dao = await arc.dao(daos[0].address)
+// or if you know the address, just create a new DAO object like this:
+const dao = arc.dao(daos[0].address)
 
 // get the DAO state
-const state  = await dao.state.toPromise()
-// the state contains the name of the DAO
-const { name }  = await dao.state.toPromise()
+const state  = await dao.state.pipe(first()).toPromise()
+// the state contains information such as the address, name or nativeToken of the DAO
+const { address, name, token }  = state
 
 // proposals
 // get a Observable of proposals that are active but not boosted
-const proposals = await dao.proposals({ active: true, boosted: false }).toPromise()
+const proposalsObservable = await dao.proposals({ active: true, boosted: false })
+const currentProposals = proposalsObservable.pipe(first()).toPromise()
```

Comment:
Why do you use `first()` here?

Reply:
`observer.toPromose()` resolves when the observer is complete. 
`proposalsObservable` is never complete - it is always looking for newly updated proposals
So what i do is create a new observable that just emits the first value and then completes (this is what `first()` does), and then check that promise...

Reply:
Oh ok, I misunderstood first, thought it gives just the first element in the list of proposals...